src/rebar3_appup_generate.erl

```diff
@@ -206,15 +206,15 @@ generate_instruction(deleted, File) ->
 
 generate_instruction(changed, ModDeps, {File, _}) ->
     {ok, {Name, List}} = beam_lib:chunks(File, [attributes, exports]),
-    Behavior = get_behavior(List),
+    Supervisor = is_supervisor(List),
     CodeChange = is_code_change(List),
```

Comment:
- [special process](http://erlang.org/doc/design_principles/spec_proc.html#id74399)
- [Mod:system_code_change-4](http://erlang.org/doc/man/sys.html#Mod:system_code_change-4)

If a module export system_code_change or code_change, it shoud have instruction that is `{update, Module, {advanced, Extra}}`. [See also](https://github.com/lrascao/rebar3_appup_plugin/pull/4)
However, it is not related to behaviour. (it related to `is_code_change/1`)

Modules except these and supervisor want `add_module` or `delete_module`, `load_module`.